core/src/components/input/input.ionic.scss

```
@@ -28,6 +37,10 @@
   color: var(--ionic-color-neutral-600, #535353);
 }
 
+:host(.has-focus.ion-valid) .helper-text {
+  color: #{$ionic-color-success-800};
```

```suggestion
  color: $ionic-color-success-800;
```

~ Sass variables do not need to be escaped: `#{ }` here.